### DISCUSSION TITLE
Dai Stuck in wormhole-Eth to Solana

### DISCUSSION THREAD

**Discussion by justtryingto** [wormhole-foundation/wormhole#244] [Problems / bugs] [answered]
Hi, My dai is stuck in the wormhole 
tx id is 0x783f7128232f0b98ee30efe410294ea95a4a9e180dd37e66dc4383a969a84193
My solana wallet is D3qE6Y9zJGCALarjVMx6tZ35F7QvKHzKu4p9Hainwavr

Thanks

**Comment by hendrikhofstadt** [comment, marked as answer]
Completed in https://explorer.solana.com/tx/42GcvGZAmU7wwZv3iraw315SjaBrsWAZFV8Sf98CacPxALXdtiMAzkaPYYuJwyTzubCAet3da34EF36DTnxaHa1n

**Reply by justtryingto** [reply]
Thank you!
